scripts/copy_to_distributed_table.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+format='text'
+options='OIDS false, FREEZE false'
+schema='public'
+
+while getopts ':BCc:d:e:Hh:n:q:T' o; do
+	case "${o}" in
+		B)
+			format='binary'
+			;;
+		C)
+			format='csv'
+			;;
+		c)
+			encoding=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			options="${options}, ENCODING '${encoding}'"
+			;;
+		d)
+			delimiter=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			options="${options}, DELIMITER '${delimiter}'"
+			;;
+		e)
+			escape=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			options="${options}, ESCAPE '${escape}'"
+			;;
+		H)
+			options="${options}, HEADER true"
+			;;
+		h)
+			usage
+			;;
+		n)
+			null=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			options="${options}, NULL '${null}'"
+			;;
+		q)
+			quote=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			options="${options}, QUOTE '${quote}'"
+			;;
+		s)
+			schema=`echo ${OPTARG} | sed s/\'/\'\'/g`
+			;;
+		T)
+			format='text'
+			;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+options="${options}, FORMAT ${format}"
+filename=$1
+tablename=$2
+
+if [ -z "${filename}" ] || [ -z "${tablename}" ]; then
+	usage
+fi
+
+filename=`echo ${filename} | sed s/\'/\'\'/g`
+facadename=`echo "${tablename}_copy_facade" | sed s/\"/\"\"/g`
```

Comment:
Are we protected against multiple copies into the same table? I noticed in the previous change we also held a lock. 
In this case, the temp table is the same, so running the script twice would error out?